configure

```diff
@@ -192,7 +192,7 @@ main() {
     CLEANALLFILES="${CLEANALLFILES} src"
     mkdir -p src/github.com/jmmv
     [ -h src/github.com/jmmv/sourcachefs ] \
-        || ln -hfs ../../.. src/github.com/jmmv/sourcachefs \
+        || ln -fs ../../.. src/github.com/jmmv/sourcachefs \
```

Comment:
Does this work properly if the symlink already exists? `-h` prevents `ln` from following that symlink, and without it, I recall getting an extra subdirectory in the given path if it happened to already exist as a symlink.

If I'm reading things correctly, Linux's `-h` equivalent is `-n`, so maybe we need to do this on a case by case basis. Or add an explicit `rm -f` of the symlink before doing the `ln`.

Reply:
I thought since there is `[ -h src/github.com/jmmv/sourcachefs ]`, we don't need the option because `ln` doesn't get executed if the symbolic exists. Am I miss something?

As far as the Linux counterpart, yes, you're right. we can use `-n`.

Reply:
I guess you are right.